app/decorators/lesson_decorator.rb

```diff
@@ -9,6 +9,7 @@ def title
   end
 
   def github_url
+    # Change lesson pull location in services/lesson_content_importer
```

Comment:
Maybe we should change the name of this method to stop the confusion?

Reply:
I'm not sure what this decorator does